theme/n21/static/scripts/n21-blog.js

```diff
@@ -0,0 +1,35 @@
+const initialized = false;
+$(document).ready(() => {
+	if (initialized) {
+		return;
+	}
+
+	function sanitize(input) {
+		const output = input.replace(/<script[^>]*?>.*?<\/script>/gi, '')
+			.replace(/<style[^>]*?>.*?<\/style>/gi, '')
+			.replace(/<![\s\S]*?--[ \t\n\r]*>/gi, '')
+			.replace(/&nbsp;/g, '');
+		return output;
+	}
+
+	const contentKey = document.querySelector('script[data-contentkey]').dataset.contentkey;
+
+	$.ajax({
+		url: 'https://blog.niedersachsen.cloud/ghost/api/v2/content/pages/slug/startseite-nbc/',
+		data: {
+			key: contentKey,
```

Comment:
never expose secrets, this should be implemented in the clients backend, by design, the frontend should only request it's node application that handles things like that.

Reply:
but: still load the content dynamically :)

Reply:
no, this key is by design public. We already implemented it the same way in nuxt